### PR TITLE
finished login/signup pages

### PR DESCRIPTION
**Pull request by bryanyuan1**
finished the signup front-end.

### CODE REVIEW

**Comment by djuanit0x** [comment]
Going to review this tonight

**Comment by bryanyuan1** [comment]
Thank you for your time!

File: client/src/App.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import React from 'react';
 import { BrowserRouter, Route, Switch } from 'react-router-dom';
-import Navbar from './components/Navigation';
+import Navbar from './components/navigation';
```
Comment (djuanit0x) [inline client/src/App.js]: I can't compile the app. I can run the app when replacing `'./components/navigation'` with `'./components/Navigation'`

File: client/src/components/General/Signup/styled.jsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  flex-direction: column;
+  align-items: center;
+`;
+
+const dividerStyle = {
+  color: '#6C6C6C',
+  fontFamily: 'Quicksand',
+  fontStyle: 'normal',
+  fontWeight: 'bold',
+  fontSize: '13px',
```
Comment (djuanit0x) [inline client/src/components/General/Signup/styled.jsx]: Don't think we need the divider style for the signup page

File: client/src/components/General/Signup/Component.jsx [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+import { Col, Row } from '../../../lib/Grid';
+import Button from '../../../lib/Button';
+import TextField from '../../../lib/TextField';
+import FieldError from '../../../lib/Feedback/FieldError';
+import { Margin } from '../../../Styles/utils';
+import { Logo, Brand } from '../../../Styles/bases/Identity';
+import { PATH_FORGOT_PASSWORD } from '../../../static/constants/appPath';
+import { Form, Field } from 'react-final-form';
+import {
+  dividerStyle,
```
Comment (djuanit0x) [inline client/src/components/General/Signup/Component.jsx]: We can get rid this one too for the signup page

File: client/src/components/General/Signup/Component.jsx [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+  passwordLength,
+  passwordRequired
+} from '../../../utils/validationRules';
+
+import { Divider } from 'semantic-ui-react';
+class Signup extends Component {
+  state = {
+    signup: true,
+    emailAddress: '',
+    password: ''
```
Comment (djuanit0x) [inline client/src/components/General/Signup/Component.jsx]: Another state for the confirm password field? See the problem[ here](https://gyazo.com/785a40a4c53ea794d846d37ddb1737be)

File: client/src/App.js (hunk 1 above)
Comment (bryanyuan1) [reply client/src/App.js]: It might be a folder naming problem.

File: client/src/components/General/Signup/Component.jsx (hunk 4 above)
Comment (bryanyuan1) [reply client/src/components/General/Signup/Component.jsx]: Yea I will add another state for that. Thanks!

Comment (bryanyuan1) [reply client/src/components/General/Signup/Component.jsx]: Where did you update the state when we enter into the input fields? I tried to look for setState({...}) but I did not find any.

File: client/src/components/General/Signup/Component.jsx [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+                          )}
+                        />
+                      </Col>
+                    </Row>
+                  </Margin>
+                  <Margin top={25} bottom={15}>
+                    <Row justify="center">
+                      <Col xs={12} lg={12}>
+                        <Field
+                          name="password"
```
Comment (djuanit0x) [inline client/src/components/General/Signup/Component.jsx]: NVM :) Just changed name field to `"confirmPassword"` , and it should fix the issue

Comment (bryanyuan1) [reply client/src/components/General/Signup/Component.jsx]: Cool thanks

**Pull request closed without merging**